ios/App/Podfile

```diff
@@ -17,7 +17,7 @@ def capacitor_pods
   pod 'CapacitorStorage', :path => '../../node_modules/@capacitor/storage'
 end
 
-target 'App' do
+target 'Audiobookshelf' do
```

Comment:
The target changed in #371, but I missed updating the Pods config for the new target.